modules/http/src/main/scala/ch/epfl/bluebrain/nexus/commons/http/JsonLdCirceSupport.scala

```diff
@@ -27,9 +27,12 @@ trait JsonLdCirceSupport extends FailFastCirceSupport {
     * @tparam A type to encode
     * @return marshaller for any `A` value
     */
-  implicit final def marshallerHttp[A: Encoder](implicit printer: Printer = Printer.noSpaces.copy(dropNullKeys = true),
-                                                keys: OrderedKeys = OrderedKeys()): ToEntityMarshaller[A] =
-    jsonLdMarshaller.compose(implicitly[Encoder[A]].apply)
+  implicit final def marshallerHttp[A](implicit
+                                       context: Uri,
```

Comment:
This is a bit dangerous without a specific type. Could you box it in some case class Context?